src/Publisher.php

```diff
@@ -7,14 +7,44 @@
  */
 class Publisher extends Request
 {
+    /**
+    * The result of the call to basic_publish. 
+    * Assumed to be successful unless overridden with 
+    * a basic.return or basic.nack. Only valid when in 
+    * confirm_select mode
+    */
+    private $publish_result = null;
 
     /**
      * @param string  $routing
      * @param Message $message
      * @throws Exception\Configuration
      */
-    public function publish($routing, $message)
+    public function publish($routing, $message, $mandatory=false)
```

Comment:
Please add a space between variable and value
```suggestion
    public function publish($routing, $message, $mandatory = false)
```